WMIAdventure/frontend/src/js/components/global/molecules/Navbar/Navbar.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Media from "react-media";
+import {desktop, mobile} from "../../../../utils/globals";
+import menuIcon from '../../../../../assets/icons/menu2.svg';
+import homeIcon from '../../../../../assets/icons/home.svg'
+import Nav from "./styled-components/Nav";
+import Back from "./styled-components/Back";
+import Div from "./styled-components/Div";
+import NavButton from "./styled-components/NavButton";
+import Link from "./styled-components/Link";
+
+/**
+ * Props:
+ * - backLink
+ * - label
+ */
+class Navbar extends React.Component {
+
+    showMenuHandler = () => {
+
+    }
+
+    mobileNavbar = () => {
+        return (
+            <Nav>
+                <Div>
+                    <Back as={Link} to={this.props.backLink}/>
+                    {this.props.label}
+                </Div>
+                <Div>
+                    <Link to={'/'}>
+                        <NavButton image={homeIcon}/>
+                    </Link>
```

Comment:
Tu bym zrobił jednym elementem jako link, tak jak jest Back zrobione, nie jako button w linku. Według walidatora to jest zła praktyka. 
![image](https://user-images.githubusercontent.com/51495505/144294552-6cab610e-c6a3-4211-8ad2-51553422a7da.png)